app/boards/shields/dumbpad/dumbpad.zmk.yml

```diff
@@ -0,0 +1,11 @@
+file_format: "1"
```

Comment:
Another one of these .zmk.ymls for the oled_top variant I think

Reply:
Done.